src/commonMain/kotlin/fr/acinq/lightning/serialization/v3/ChannelState.kt

```diff
@@ -397,7 +398,7 @@ internal data class Commitments(
                     fr.acinq.lightning.channel.PartiallySignedSharedTransaction(
                         fr.acinq.lightning.channel.SharedTransaction(null, InteractiveTxOutput.Shared(0, commitInput.txOut.publicKeyScript, localCommit.spec.toLocal, localCommit.spec.toRemote), listOf(), listOf(), listOf(), listOf(), 0),
                         // We must correctly set the txId here.
-                        TxSignatures(channelId, commitInput.outPoint.hash, listOf()),
+                        TxSignatures(channelId, TxId(commitInput.outPoint.hash), listOf()),
```

Comment:
Nit² (same in v2):
```suggestion
                        TxSignatures(channelId, commitInput.outPoint.txid, listOf()),
```

Reply:
Done in https://github.com/ACINQ/lightning-kmp/pull/524/commits/4876605dd6bbad560b9448991c535815b123a8a7